File: CX/ClientApp/src/servicios/ServicioParticipantesEquipoTrabajo.js
import { ProcesarDatosApi } from "./ApiFetch";
const controlador = "participante";


export const InsertarParticipanteEquipoTrabajo = async (data) => {
    const url = `${controlador}/InsertarParticipanteEquipoTrabajo`;
    return await ProcesarDatosApi('POST', url, data);
}

export const ActualizarParticipante = async (data) => {
    const url = `${controlador}/ActualizarParticipante`;
    return await ProcesarDatosApi('PUT', url, data);
}

export const EliminarParticipante = async (id) => {
    const url = `${controlador}/EliminarParticipante?idParticipante=${id}`;
    return await ProcesarDatosApi('DELETE', url);
}


export const ObtenerParticipantePorID = async (id) => {
    const url = `${controlador}/ObtenerParticipantePorID/${id}`;
    return await ProcesarDatosApi('GET', url);
}

export const ObtenerEquipoTrabajoPorIdTaller = async (id) => {
    const url = `${controlador}/ObtenerEquipoTrabajoPorIdTaller/${id}`;
    return await ProcesarDatosApi('GET', url);
}

File: BussinesLogic/Servicios/ServicioParticipantesEquipoTrabajo.cs
using DataAccess.StoredProcedures;
using Dominio.Dto;
using Dominio.Entiti;
using Dominio.Interfaces.Aplicacion.ParticipantesEquipoTrabajo;
using Dominio.Interfaces.Infraestructura.BaseDatos;
using System;
using System.Collections.Generic;
using System.Text;
using System.Threading.Tasks;

namespace BussinesLogic.Servicios
{
    public class ServicioParticipantesEquipoTrabajo : IServicioParticipantesEquipoTrabajo
    {
        public readonly IRepositorioParticipantesEquipoTrabajo repositorioParticipantesEquipoTrabajo;

        public ServicioParticipantesEquipoTrabajo(IRepositorioParticipantesEquipoTrabajo repositorioParticipantesEquipoTrabajo)
        {
            this.repositorioParticipantesEquipoTrabajo = repositorioParticipantesEquipoTrabajo;
        }

        public async Task<DtoRespuestaSP> InsertarParticipanteEquipoTrabajo(EntitiParticipanteEquipoTrabajo entitiParticipante)
        {
            return await this.repositorioParticipantesEquipoTrabajo.InsertarParticipanteEquipoTrabajo(entitiParticipante);
        }

        public async Task<DtoRespuestaSP> ActualizarParticipante(EntitiParticipanteEquipoTrabajo entitiParticipanteEquipoTrabajo)
        {
            return await this.repositorioParticipantesEquipoTrabajo.ActualizarParticipante(entitiParticipanteEquipoTrabajo);
        }

        public async Task<DtoParticipanteEquipoTrabajo> ObtenerParticipantePorID(int idParticipante)
        {
            return await this.repositorioParticipantesEquipoTrabajo.ObtenerParticipantePorID(idParticipante);
        }

        public async Task<List<DtoParticipanteEquipoTrabajo>> ObtenerEquipoTrabajoPorIdTaller(int idTallerCoCreacion)
        {
            return await this.repositorioParticipantesEquipoTrabajo.ObtenerEquipoTrabajoPorIdTaller(idTallerCoCreacion);
        }
        public async Task<DtoRespuestaSP> EliminarParticipante(int idParticipantesEquipoTrabajo)
        {
            return await this.repositorioParticipantesEquipoTrabajo.EliminarParticipante(idParticipantesEquipoTrabajo);
        }

    }
}
